pkg/osvscanner/osvscanner.go

```diff
@@ -62,18 +62,11 @@ var NoPackagesFoundErr = errors.New("no packages found in scan")
 //nolint:errname,stylecheck // Would require version major bump to change
 var VulnerabilitiesFoundErr = errors.New("vulnerabilities found")
```

Comment:
Could you leave a comment here saying that this includes uncalled vulns, and license violations? The name is misleading but I get that changing it would require a major version bump change.

Reply:
Done.